apps/mobile/src/components/sheets/notify-user-sheet.layout.tsx

```diff
@@ -67,11 +71,10 @@ export function NotifyUserSheetLayout({
         </Box>
         <Button
           onPress={onNotify}
-          icon={<BellAlarmIcon color={theme.colors[getButtonTextColor('default')]} />}
           buttonState="default"
           title={t({
-            id: 'notify_user.button',
-            message: 'Notify me',
+            id: 'notify_user.interested_button',
+            message: "I'm interested!",
```

Comment:
I don't think it matters here but this[ should have no](https://trustmachines.slack.com/archives/C05114YP8CV/p1733744094907799?thread_ts=1733729747.621429&cid=C05114YP8CV) `!`